### PR TITLE
fix(key-card): fix errors in key-card drawing code

### PR DESCRIPTION
**Pull request by LeoKotschenreuther**
* adds more examples for key cards to download to the web-demo module
* fixes bugs discovered through these examples

Ticket: BG-64432

## Test instructions

1. Check out this branch locally
2. Run `yarn install` in the root of the repository
3. `cd modules/web-demo`
4. `yarn dev`
5. Go to http://localhost:8080/keycard
6. Download all three types of key cards and compare to the ones attached to this PR.

[BitGo Keycard for Hello World - polygon.pdf](https://github.com/BitGo/BitGoJS/files/10705911/BitGo.Keycard.for.Hello.World.-.polygon.pdf)
[BitGo Keycard for Hello World - ethereum.pdf](https://github.com/BitGo/BitGoJS/files/10705914/BitGo.Keycard.for.Hello.World.-.ethereum.pdf)
[BitGo Keycard for Hello World - litecoin.pdf](https://github.com/BitGo/BitGoJS/files/10705916/BitGo.Keycard.for.Hello.World.-.litecoin.pdf)

### CODE REVIEW

**Review by sachushaji — COMMENTED** [review]
image
The Tss wallets had encrypted wallet password as item `D` but should be `B` I guess?

**Comment by LeoKotschenreuther** [comment]
> image
> 
> The Tss wallets had encrypted wallet password as item `D` but should be `B` I guess?

It's the same behavior on the key cards generated by the UI. The order is Box A, Box D, Box B and Box C. I think it was changed when we introduced key cards for ETH TSS wallets because it meant that less input would be broken across multiple pages.

**Comment by sachushaji** [comment]
> > image
> > The Tss wallets had encrypted wallet password as item `D` but should be `B` I guess?
> 
> It's the same behavior on the key cards generated by the UI. The order is Box A, Box D, Box B and Box C. I think it was changed when we introduced key cards for ETH TSS wallets because it meant that less input would be broken across multiple pages.

Ohh okay, Didn't notice this.

**Review by sachushaji — APPROVED** [review]

File: modules/web-demo/src/components/KeyCard/index.tsx [hunk 1]
```diff
@@ -29,6 +45,230 @@ async function downloadKeycard() {
...
+      },
+      {
+        from: 'bitgo',
+        to: 'user',
+        publicShare:
+          '02a34344aea7a9d0495229a2da44f6bca37cb6433e16ed77e3c251b22188bd3a908448bf66517a9614f23f03ff3b39b8d0e5bba8d2539150e88e115035cfe8fa03',
```
[the remark this answers is not in the document]
Comment (LeoKotschenreuther) [reply modules/web-demo/src/components/KeyCard/index.tsx]: Moved to a separate file

**Review by sachushaji — APPROVED** [review]
LGTM, Thanks.

**Review by zahin-mohammad — APPROVED** [review]
lgtm

**Pull request merged**